### ISSUE TITLE
Предложение

### ISSUE THREAD

**Issue by A-platinum** [shamhi/YesCoinBot#32]
Предлагаю ввести автоматический клайм ежедневных наград.

**Comment by iskizreal** [comment]
Да, было бы отлично, добавить ежедневный клейм за чекин. Он там правда не большой, но все же)
